packages/shared/components/Drawer.svelte

```diff
@@ -21,13 +20,13 @@
 
     $: darkModeEnabled = $appSettings.darkMode
 
-    export let dimLength = 160
     export let opened = false
     export let fromRight = false
     export let classes = ''
     export let fullScreen = false
     export let preventClose = false
     export let zIndex = 'z-30'
+    export let onClose = (): void => {}
```

Comment:
Here you initialize it as an empty function. To type it, it would be better to do the following:

```suggestion
    export let onClose: () => void
```

Reply:
Here It's needed to be initialized since the drawer can be instanced without onClose, in this case onClose will remain undefined and we get next type error when it's called from the component itself
```error
Uncaught (in promise) TypeError: onClose is not a function
    at proxy.<anonymous> (Drawer.svelte:134:1)
```